index.html

```diff
@@ -1,33 +1,31 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <link rel="stylesheet" href="/src/css/main.css">
-        <link rel="stylesheet" href="/src/css/typography.css">
+  <head>
+    <!-- Use type="module" so can import Header.js as an ES6 module -->
+    <script src="./src/components/Header.js" type="module"></script>
+    <script src="./src/js/main.js" type="module"></script>
+    <link rel="stylesheet" href="./src/css/main.css" />
+    <link rel="stylesheet" href="./src/css/typography.css" />
+  </head>
 
-    </head>
+  <body>
+    <!-- Used by Header.js -->
+    <header></header>
```

Comment:
This tag is necessary so document.QuerySelector("header") can be used in `main.js`